### PR TITLE
Rearch, run tests, rerender

### PR DESCRIPTION
**Pull request by bollwyvl**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Just tried to install this on 3.7, and no :game_die: due to `dataclasses`. `importlib_resources` apparently is fine on 3.7, but went ahead and put that behind a selector as well. `nbconvert` is also no longer a runtime dependency, and `setuptools` is only needed for `ipython` usage, but went ahead and left it in.

Have also:
- ~~added the test suite~~ NB: need to work out an upstream symlink issue for windows
- fixed up the `entry_points`
  - ~~and test them~~ NB: sort of: notified upstream of "dead" entry_point
- updated some more metadata

We'll see how it goes!

### CODE REVIEW

**Comment by bollwyvl** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
@conda-forge-admin, please rerender

**Comment by bollwyvl** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -9,35 +9,49 @@ source:
...
 build:
-  number: 0
-  noarch: python
+  skip: true  # [py<34]
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  entry_points:
+    - importnb-install = importnb.utils.ipython:install
+    - importnb-uninstall = importnb.utils.ipython:uninstall
+    - nbdoctest = importnb.utils.nbdoctest:_test
```
Comment (ocefpaf) [inline recipe/meta.yaml:18]: By specifying the entry-points I don't believe you need to re-arch.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -9,35 +9,49 @@ source:
...
     - pytest-runner
     - wheel >=0.31.0
     - twine >=1.11.0
   run:
-    - python >=3
+    - python
     - setuptools
-    - dataclasses
-    - nbconvert
-    # importlib_resources will be [not py>36] and this will be arch again :-(
```
Comment (ocefpaf) [inline recipe/meta.yaml:28]: Is this true now? If so there is nothing we can do :-(

**Review by ocefpaf — APPROVED** [review]

**Comment by ocefpaf** [comment]
Not sure what is going on with Travis-CI, let's merge and see if that persists on master. Thanks @bollwyvl!

**Comment by bollwyvl** [comment]
Thanks!

The rearch ended up being due to data classes,

https://github.com/conda-forge/dataclasses-feedstock/blob/c70d555ddaffab9350414138129a3cc5e718686d/recipe/meta.yaml#L21

importlib_resources has a forward port.

I suppose we could make a _dataclasses `noarch: generic` metapackage which
had no dependencies in py37... Sneaky...

**Pull request merged**